moped-editor/src/views/projects/projectView/NoteInput.js

```diff
@@ -35,22 +41,32 @@ const useStyles = makeStyles((theme) => ({
     color: theme.palette.secondary.dark,
   },
   contentEditable: {
-    height: "300px",
     width: "100%",
-    padding: "8px 8px",
-    border: "1px solid #fff"
+    padding: "16px 16px",
```

Comment:
these jumped out at me this time around. Could you update these to use `theme.spacing(2)` so this uses our MUI default scaling factors?